sources/packages/web/src/assets/css/formio-shared.scss

```diff
@@ -428,6 +428,15 @@
   min-height: $image-min-height;
 }
 
+.person-adding-file-to-folder-background {
```

Comment:
line 433 to 437 are same across all the css classes for the images.

We can declare once common css class and extend from it using @extend .common-class

Reply:
ok. Done that.

Reply:
resolving as per our discussion.